src/Parser/Hunter/BeastMastery/Modules/Items/QaplaEredunWarOrder.js

```diff
@@ -44,11 +47,57 @@ class QaplaEredunWarOrder extends Analyzer {
       this.wastedKillCommandReductionMs += COOLDOWN_REDUCTION_MS;
     }
   }
+
+  wastedKillCommandCDRPercent() {
+    return this.wastedKillCommandReductionMs / (this.wastedKillCommandReductionMs + this.effectiveKillCommandReductionMs);
+  }
+
+  averageEffectiveCDR() {
+    return (this.effectiveKillCommandReductionMs / ((this.wastedKillCommandReductionMs + this.effectiveKillCommandReductionMs) / (COOLDOWN_REDUCTION_MS / 1000)));
+  }
+
+  get wastedSuggestionThreshold() {
+    return {
+      actual: this.averageEffectiveCDR(),
+      isLessThan: {
+        minor: 2.5,
+        average: 2.25,
+        major: 2,
+      },
+      style: 'decimal',
+    };
+  }
+
+  get killerCobraThreshold() {
+    return {
+      actual: this.combatants.selected.hasTalent(SPELLS.KILLER_COBRA_TALENT.id),
+      isEqual: true,
+      style: 'boolean',
+    };
+  }
+
+  suggestions(when) {
+    const spellName = this.combatants.selected.hasTalent(SPELLS.DIRE_FRENZY_TALENT) ? SPELLS.DIRE_FRENZY_TALENT.name : SPELLS.DIRE_BEAST.name;
+    when(this.killerCobraThreshold).addSuggestion((suggest) => {
+      return suggest(<Wrapper>Due to the <SpellLink id={SPELLS.KILL_COMMAND.id} icon /> reduction capabilities of both <ItemLink id={ITEMS.QAPLA_EREDUN_WAR_ORDER.id} icon /> and <SpellLink id={SPELLS.KILLER_COBRA_TALENT.id} icon />, using them together is generally not recommended. </Wrapper>)
+        .icon(ITEMS.QAPLA_EREDUN_WAR_ORDER.icon)
+        .actual(`You had both Qa'pla, Eredun War Order equipped and talented Killer Cobra`)
+        .recommended(`Only one or the other is recommended`)
+        .staticImportance(SUGGESTION_IMPORTANCE.MINOR);
+
+    });
+    when(this.wastedSuggestionThreshold).addSuggestion((suggest, actual, recommended) => {
+      return suggest(<Wrapper>Your average cast of {spellName} reduced <SpellLink id={SPELLS.KILL_COMMAND.id} icon /> by less than {recommended} seconds. Try and optimise this legendary by making sure to utilise it's cooldown reduction utility better. </Wrapper>)
+        .icon(ITEMS.QAPLA_EREDUN_WAR_ORDER.icon)
+        .actual(`${(actual).toFixed(2)} average seconds of CDR per ${spellName} cast`)
+        .recommended(`>${recommended}s of CDR per ${spellName} is recommended`);
```

Comment:
I'd suggest shortening this to `>${recommended} sec is recommended` since you already have "of cdr per spellname" in the `actual` text.